platforms/common/TGFontConverter.h

```diff
@@ -0,0 +1,24 @@
+//
```

Comment:
Yup will do. And yes it has no dependency on harfbuzz updates. I had just thought, that PR will go in first :D.

Also we should wait on `0.8.0` to be released before we merge this or any other PR in.